File: result/translate/chatgpt/python2java/java/MINIMUM_OPERATION_MAKE_ELEMENTS_EQUAL_ARRAY.java
public static int f_gold(int[] arr, int n) {
    HashMap<Integer, Integer> Hash = new HashMap<>();
    for (int i = 0; i < n; i++) {
        Hash.put(arr[i], Hash.getOrDefault(arr[i], 0) + 1);
    }
    int max_count = 0;
    for (int i : Hash.keySet()) {
        if (max_count < Hash.get(i)) {
            max_count = Hash.get(i);
        }
    }
    return n - max_count;
}

File: result/repair/starchat/java2cpp/c++_repair/MINIMUM_OPERATION_MAKE_ELEMENTS_EQUAL_ARRAY.cpp
int f_gold ( int arr [ ], int n ) {
    unordered_map<int,int> mp;
    for (int i=0;i<n;i++) mp[arr[i]]++;
set<int> ans = 0;
    for (auto x:mp) if (x.second==1) ans++;
    return n-ans;
}
